src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/appsettings.json

```diff
@@ -1,36 +1,37 @@
 {
-////#if (IndividualB2CAuth)
-//  "AzureAdB2C": {
-//    "Instance": "https:////login.microsoftonline.com/tfp/",
-//    "ClientId": "11111111-1111-1111-11111111111111111",
-//    "CallbackPath": "/signin-oidc",
-//    "Domain": "qualified.domain.name",
-//    "SignUpSignInPolicyId": "MySignUpSignInPolicyId",
-//    "ResetPasswordPolicyId": "MyResetPasswordPolicyId",
-//    "EditProfilePolicyId": "MyEditProfilePolicyId"
-//  },
-////#elseif (OrganizationalAuth)
-//  "AzureAd": {
-//#if (MultiOrgAuth)
-//    "Instance": "https:////login.microsoftonline.com/common",
-//#elseif (SingleOrgAuth)
-//    "Instance": "https:////login.microsoftonline.com/",
-//    "Domain": "qualified.domain.name",
-//    "TenantId": "22222222-2222-2222-2222-222222222222",
-//#endif
-//    "ClientId": "11111111-1111-1111-11111111111111111",
-//    "CallbackPath": "/signin-oidc"
-//  },
-//#endif
-////#if (IndividualLocalAuth)
-//  "ConnectionStrings": {
-////#if (UseLocalDB)
-//    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-Company.WebApplication1-53bc9b9d-9d6a-45d4-8429-2a2761773502;Trusted_Connection=True;MultipleActiveResultSets=true"
-////#else
-//    "DefaultConnection": "DataSource=app.db"
-//#endif
-//  },
-//#endif
+  ////#if (IndividualB2CAuth)
+  //  "AzureAdB2C": {
+  //    "Instance": "https:////login.microsoftonline.com/tfp/",
+  //    "ClientId": "11111111-1111-1111-11111111111111111",
+  //    "CallbackPath": "/signin-oidc",
+  //    "Domain": "qualified.domain.name",
+  //    "SignUpSignInPolicyId": "MySignUpSignInPolicyId",
+  //    "ResetPasswordPolicyId": "MyResetPasswordPolicyId",
+  //    "EditProfilePolicyId": "MyEditProfilePolicyId"
+  //  },
+  ////#elseif (OrganizationalAuth)
+  //  "AzureAd": {
+  //    "Instance": "https:////login.microsoftonline.com/",
+  //    "Autority": "{Instance}{TenantId}/v2.0",
+  //#if (MultiOrgAuth)
+  //    "TenantId": "organizations",
```

Comment:
Why do we need this now? MultiOrg is kind of it's own beast and it used to point to `https:////login.microsoftonline.com/common` would this now point to `https:////login.microsoftonline.com/organizations`?

Asa general principle we don't introduce concepts in the template if we don't absolutely need to, so could this go back to the way it was before?

Reply:
So in Azure AD v2.0, common means AAD+MSA whereas in Azure AD v1.0 it used to mean "AAD multi-org. 
To say AAD multi-org in "Azure AD v2.0", one needs to use organizations. I proposed this one, in order to not change the behavior of the template (by using organizations with a v2.0 template, it behaves as common for a v1.0 template).